source/tinyengine_debug_utilities.h

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <windows.h>
+
+static void
+Print(char* Format, ...)
+{
+    static char Buffer[1024];
+    va_list ArgumentList;
+    va_start(ArgumentList, Format);
+    vsprintf_s(Buffer, sizeof(Buffer), Format, ArgumentList);
+    va_end(ArgumentList);
+    OutputDebugStringA(Buffer);
```

Comment:
Since OutputDebugString is platform specific we should not directly call it here, as if someone includes this file on linux it wont work